lib/evmone_precompiles/bls.cpp

```diff
@@ -17,7 +17,7 @@ constexpr auto MAIN_SUBGROUP_ORDER =
 
 /// Validates p1 affine point. Checks that point coordinates are from the BLS12-381 field and
 /// that the point is on curve. https://eips.ethereum.org/EIPS/eip-2537#abi-for-g1-addition
-[[nodiscard]] std::optional<blst_p1_affine> validate_point(
+[[nodiscard]] std::optional<blst_p1_affine> validate_point_g1(
```

Comment:
```suggestion
[[nodiscard]] std::optional<blst_p1_affine> validate_p1(
```

I think we can skip the "point" name, and use `p1` or `e1` if there is no subgroup check.

Reply:
done